### ISSUE TITLE
[Pi Zero W] Unable to initialize ws281x library.

### ISSUE THREAD

**Issue by jesusignazio** [hyperion-project/hyperion#753]
I'm trying to use a **Raspberry Pi Zero W** with a **ws2812b** LEDs following the 3-wires configuration.

It worked previously with a RPi2 with the exactly same configuration.

_hyperion.config.json_
```
"device" :
	{
		 "name"  : "MyPi",
                 "type"  : "ws281x",
                 "colorOrder" : "grb",
                 "leds"  : 130,
                 "gpio"  : 18,
                 "dmanum"  : 5,
                 "freq"  : 800000,
                 "pwmchannel" : 0
	},
```

_systemctl status hyperion_
```
May 05 20:18:05 ambilight hyperiond[970]: }
May 05 20:18:05 ambilight hyperiond[970]: whiteAlgorithm ::
May 05 20:18:05 ambilight hyperiond[970]: Unable to initialize ws281x library.
May 05 20:18:05 ambilight hyperiond[970]: terminate called after throwing an instance of 'int'
May 05 20:18:05 ambilight systemd[1]: hyperion.service holdoff time over, scheduling restart.
May 05 20:18:06 ambilight systemd[1]: Stopping Hyperion Systemd service...
May 05 20:18:06 ambilight systemd[1]: Starting Hyperion Systemd service...
May 05 20:18:06 ambilight systemd[1]: hyperion.service start request repeated too quickly, refusing to start.
May 05 20:18:06 ambilight systemd[1]: Failed to start Hyperion Systemd service.
May 05 20:18:06 ambilight systemd[1]: Unit hyperion.service entered failed state.
```

_cat /proc/cpuinfo_
```
processor	: 0
model name	: ARMv6-compatible processor rev 7 (v6l)
BogoMIPS	: 697.95
Features	: half thumb fastmult vfp edsp java tls 
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xb76
CPU revision	: 7

Hardware	: BCM2835
Revision	: 9000c1
```

Thanks all.

**Comment by penfold42** [comment]
Should be fixed in #755 which updates the underlying submodule

**Comment by jesusignazio** [comment]
Solved. Thanks.
